Makefile

```diff
@@ -73,6 +76,7 @@ build: node_modules ## Builds the app
 
 	rm -rf dist
 
+	npm run build --workspace=packages/mattermost-components
```

Comment:
Perhaps we should have `npm run build` also run the build for each workspace? 0/5 on that though since the other ones likely work differently at the moment

Reply:
We should eventually but they don't build for me at the moment.

Reply:
Oh yeah, I meant doing this automatically like have the build script for the web app be `npm build --workspaces && <whatever else it does now>` instead of having that be in the Makefile